src/rationale/fork_choice_das.md

```diff
@@ -0,0 +1,39 @@
+# Fork Choice Rule with Data Availability Sampling
+
+- [Preamble](#preamble)
+- [Invalid vs Unavailable](#invalid-vs-unavailable)
+- [Scenarios](#scenarios)
+
+## Preamble
+
+Tendermint provides finality under an honest 2/3 of stake assumption. It is one of several ["BFT" consensus protocols](https://arxiv.org/abs/1807.04938) (also known as "classical" consensus protocols). Under that assumptions, new _valid_ blocks are immediately and forever final as soon as 2/3 of stake commits to the block. Therefore, under that assumption, Tendermint is fork-free.
+
+Contemporary blockchains support full nodes (which are secure under no assumption on stake honesty) and light nodes (which are secure under an honest majority of stake assumption). LazyLedger is unique in [supporting light nodes with stronger security guarantees](./../specs/node_types.md#node-type-definitions):
```

Comment:
> and light nodes (which are secure under an honest majority of stake assumption)

I think the security of light clients is a bit more nuanced. See for instance: https://github.com/tendermint/spec/tree/b270ab8d15410ab4e70e3835ef2bbd3bbf9c48d7/spec/light-client/attacks

Reply:
Ah, maybe this isn't clear, but "secure" here means "doesn't accept invalid blocks," not "safe" (aka "consistent"). The light client doc you linked to is about consensus safety failures, which is of course important, but detecting equivocation is orthogonal to DAS.

Reply:
Clarify in d76e65b.